src/main/java/io/fabric8/maven/docker/access/ContainerCreateConfig.java

```diff
@@ -57,6 +57,12 @@ public ContainerCreateConfig environment(String envPropsFile, Map<String, String
                 String value = entry.getValue();
                 if (value == null) {
                     value = "";
+                } else if(value.contains("+")) {
```

Comment:
wold prefer a `startsWith("+$")` here because otherwise its far too unspecific. (Think about values which can validly contain `+` characters)

Even better, use a regexp `^\+\$\{.*\}$`. This would really only match our use case.